src/main/java/com/koczy/kurek/mizera/thesisbrowser/controller/LdaController.java

```diff
@@ -27,6 +30,12 @@ public LdaController(ILdaService ldaService) {
         return ldaService.run();
     }
 
+    @RequestMapping(value = "/api/similarThesesAmongFromFilter", method = RequestMethod.GET)
+    public ResponseEntity<List<ThesisResponse>> getSimilarThesesAmongFromFilter(@RequestBody ArrayList<Integer> exemplaryTheses,
+                                                                                @RequestBody ThesisFilters thesisFilters) {
+        return ldaService.getSimilarThesesFromFilter(exemplaryTheses, thesisFilters);
```

Comment:
Proponuje tez zrobić demo bo to jest button dostępny dla wszystkich i teraz jak ktoś kliknie na demo to będzie trochę spina